src/reanimated2/initializers.ts

```diff
@@ -14,13 +14,13 @@ const SHOULD_BE_USE_WEB = shouldBeUseWeb();
 const IS_CHROME_DEBUGGER = isChromeDebugger();
 
 // callGuard is only used with debug builds
-export function callGuardDEV<T extends Array<unknown>, U>(
-  fn: (...args: T) => U,
-  ...args: T
-): void {
+export function callGuardDEV<Args extends unknown[], ReturnValue>(
+  fn: (...args: Args) => ReturnValue,
+  ...args: Args
+): ReturnValue | void {
```

Comment:
```suggestion
): ReturnValue {
```

Reply:
unfortunately we can't :/ because of `try` block inside there is possibility that function will return nothing

Reply:
so what's the expected return value of a worklet that throws an error on the UI runtime? should we re-throw the error in the RN runtime or what?